python/ray/data/read_api.py

```diff
@@ -886,7 +887,7 @@ class   string
         ValueError: if ``mode`` is unsupported.
     """
     if meta_provider is None:
-        meta_provider = get_image_metadata_provider()
+        meta_provider = _ImageFileMetadataProvider()
```

Comment:
Nit: Should we rename `_ImageFileMetadataProvider` to `ImageFileMetadataProvider` since it is now being used outside image datasource?